README.md

```diff
@@ -89,6 +89,24 @@ Your `~/.vimrc.local` might look like this:
     highlight NonText guibg=#060606
     highlight Folded  guibg=#0A0A0A guifg=#9090D0
 
+If you don't wish to install a vim plugin from the default set of vim plugins in
+`.vimrc.bundles`, you can ignore the plugin by calling it out with `UnPlug` in
+your `~/.vimrc.bundles.local`.
+
+    " Don't install vim-scripts/tComment
+    UnPlug 'tComment'
+
+`UnPlug` can be used to install your own fork of a plugin or to install a shared
```

Comment:
In reading your code here I'm realizing that the plugins are stored in the `g:plugs` dictionary as just the plugin name, not `author/plugin-name` which I would have initially expected. This was a surprise to me. Do you think it's worth highlighting this in the summary?